### ISSUE TITLE
non-unicode `Active` selection

### ISSUE THREAD

**Issue by dm9pZCAq** [gennaro-tedesco/gh-s#16]
for now there is `\U0001F449` used but it would be nice to have option to change this somehow
for example through **config**, **flag** or **environment variable**

it would be useful for those who don't have fonts with this unicode

---
https://github.com/gennaro-tedesco/gh-s/blob/dd35856aa044749f1a857ee3ae0fd0bc21134ed2/cmd/ui.go#L76

**Comment by gennaro-tedesco** [comment]
You are completely right, I overlooked it. These days I am away from the computer but I will propose an alternative as I am back (a practical solution could be to check for fonts and default back, so that users do not have to bother with it, although I am not sure how easy it is to do programmatically).
